templates/scenarios/csharp/non-sso-tab/Components/Hello.razor

```diff
@@ -0,0 +1,42 @@
+﻿@inject MicrosoftTeams MicrosoftTeams;
+
+@if(isLoading)
+{
+    <div style="display: flex; justify-content: center; align-items: center; min-height: 100vh;">
+		<FluentProgressRing/>
+	</div>
```

Comment:
It appears that there may be some issues with the formatting of this code, others look good to me.

Reply:
Good Catch! There are some tabs instead of space. Let me update it.